opentech/apply/review/templates/review/review_detail.html

```diff
@@ -18,9 +18,17 @@ <h5>Recommendation</h5>
         <h5>Score</h5>
         <p>{{ review.score }}</p>
     </div>
+    {% if not review.for_latest %}
+        <div>
+            <h5>Review was not against the lasted version:</h5>
```

Comment:
"The Review was not added against the latest submission version"?
(or at least `s/lasted/latest`)